.github/workflows/image.yaml

```diff
@@ -91,4 +98,6 @@ jobs:
           platforms: ${{ env.IMAGE_PLATFORMS }}
           build-args: |
             GOLANG_VERSION=${{ matrix.go-version }}
+            CONTROLLER_GEN_VERSION=${{ matrix.controller-gen-version || "v0.9.2" }}
```

Comment:
The workflow is not valid. .github/workflows/image.yaml (Line: 99, Col: 23): Unexpected symbol: '"v0'. Located at position 34 within expression: matrix.controller-gen-version || "v0.9.2"

Reply:
Thanks, looks like you have to use single quotes inside the expressions.